WordPress/src/main/java/org/wordpress/android/ui/JetpackUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.ui;
+
+import org.wordpress.android.analytics.AnalyticsTracker;
+
+import java.util.HashMap;
+
+/**
+ * Wraps utility methods for Jetpack
+ */
+public class JetpackUtils {
```

Comment:
The `JetpackUtils` name carries a lot of weight for me. I'd find it confusing to find that the only method in there is a `trackWithSource` method which actually only tracks the jetpack connection stuff. Maybe we can rename the class to something like `JetpackConnectionUtils` and move some other reusable parts (if there are any) related the the Jetpack connection here? What do you think?

Reply:
I thought it could be a utility class others could reuse. I could rename it (maybe to `JetpackTrackingUtils` since it's only related to tracking)? I didn't see any other static methods for which it would make sense to move here.

Reply:
I personally prefer `JetpackConnectionUtils` because I'd again expect other things to be in `JetpackTrackingUtils` but I don't think we'll be adding them. But, this is totally up to you, you can even leave it as is if you think that's better.